### DISCUSSION TITLE
Internal links in footnotes

### DISCUSSION THREAD

**Discussion by xuhcc** [community/community#23036] [Other Feature Feedback, Questions, & Ideas]
<p>I’m using Github Pages service to generate a static website from a collection of markdown files.</p>
<p>Links in footnotes are not working as expected. They are rendered as links to markdown files, instead of links to html files.</p>
<p><strong>test1.md</strong></p>
<pre><code>TEST[^1].

[test2](test2.md). # &lt;a href="/test2.html"&gt;test2&lt;/a&gt;

[^1]: [test2](test2.md). # &lt;a href="test2.md"&gt;test2&lt;/a&gt;
</code></pre>
<p><strong>test2.md</strong></p>
<pre><code>TEST
</code></pre>
<p>Probably it is a bug in Jekyll, but I’m not sure.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Strictly speaking it is an issue with the “markdown converter”. The default used to be <a href="https://kramdown.gettalong.org/syntax.html" rel="nofollow noopener">kramdown</a>, in case you don’t have markdown defined in your _config.yml.</p>
<p>So you might dig into some other footnote plugins for Jekyll to see if any of them solve your issue.  <a href="https://github.com/orangejulius/jekyll-footnotes" rel="nofollow noopener">E.g.</a> </p>

**Comment by xuhcc** [comment]
<p>Thanks for reply! I submitted a bug report to kramdown issue tracker.</p>
<p>Tried jekyll-footnotes plugin, but it doesn’t seem to be working.</p>
